### PR TITLE
added enrollment complete indicator fixes #1293

### LINKED ISSUE #1293: Add an “Enrollment Paused” label next to the “Live” label.
Add an “Enrollment Paused” label next to the “Live” label.  
image

NEW REQUEST: Have a label on experiments have "Enrollment Paused"  - that is based off the Normandy API Pause information.

OLD REQUEST (we aren't doing this based on ability to get info from Normandy API):
This should be controlled by a Normandy folks manual click.  There is not a programmatic way for Normandy to indicate if an experiment is paused.  So would like a manual click, so we know when these transitions have been done without looking at code.

Maybe under “Archive Experiment” - but it’s not available unless you report to Benson (or are benson or shell).

### PR DESCRIPTION
**Pull request by tiftran**
Screen Shot 2019-07-19 at 10 45 15 AM

Screen Shot 2019-07-19 at 2 00 13 PM

### CODE REVIEW

**Comment by jaredkerim** [comment]
@tiftran screenshots?

**Comment by tiftran** [comment]
🤦‍♀ sorry... I had them... but i didn't include them

**Comment by uhlissuh** [comment]
is there any way we can make that filter label less long so it doesn't go to a new line? 😄 seems hard if we can't use the word "paused" since people outside of experimenter don't know what that word means?

**Comment by tiftran** [comment]
hmmmm @uhlissuh  @jaredkerim  would "Show completed enrollments" in place of "show enrollment complete experiments" make sense?

File: app/experimenter/experiments/models.py [hunk 1]
```diff
@@ -700,6 +700,10 @@ def firefox_channel_sort():
     def is_archivable(self):
         return self.status != self.STATUS_LIVE
 
+    @property
+    def is_enrollment_complete(self):
+        return self.is_paused and self.status == self.STATUS_LIVE
```
Comment (jaredkerim) [inline app/experimenter/experiments/models.py]: No model tests for this method?

**Comment by tiftran** [comment]
@jaredkerim @uhlissuh i was thinking of a diff color that was distinctly different from all the other status pills and ended up with yellow... secretly hoping yellow was different enough to the sign off orange...

Maybe a dark green might be a better idea? so that greens are reserved for live?
Screen Shot 2019-07-23 at 1 58 03 PM

**Comment by jaredkerim** [comment]
Yeah I like the dark green better it feels less confusing.  So funny a bunch of engineers trying to figure out what colours mean....

File: app/experimenter/experiments/tests/test_models.py [hunk 2]
```diff
@@ -569,6 +569,23 @@ def test_observation_dates_returns_date_string(self):
             "Jan 11, 2019 - Jan 21, 2019 (10 days)",
         )
 
+    def test_enrollment_is_complete(self):
```
Comment (jaredkerim) [inline app/experimenter/experiments/tests/test_models.py]: 👍

File: app/experimenter/experiments/tests/test_models.py [hunk 3]
```diff
@@ -569,6 +569,23 @@ def test_observation_dates_returns_date_string(self):
...
+        )
+        self.assertTrue(experiment.is_enrollment_complete)
+
+    def test_enrollment_is_not_complete(self):
+        experiment = ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_LIVE, is_paused=False
+        )
+        self.assertFalse(experiment.is_enrollment_complete)
+
+        # wrong status
```
Comment (jaredkerim) [inline app/experimenter/experiments/tests/test_models.py]: What's this doing here?

Comment (tiftran) [reply app/experimenter/experiments/tests/test_models.py]: 🤔  not sure... will remove...

**Pull request merged**